src/test/java/com/exasol/adapter/dialects/saphana/util/IntegrationTestConstants.java

```diff
@@ -0,0 +1,28 @@
+package com.exasol.adapter.dialects.saphana.util;
+
+import java.nio.file.Path;
+
+public final class IntegrationTestConstants {
+    public static final String VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION = "virtual-schema-dist-10.1.0-hana-2.1.0.jar";
+    public static final Path PATH_TO_VIRTUAL_SCHEMAS_JAR = Path.of("target", VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION);
+    public static final String SCHEMA_EXASOL = "SCHEMA_EXASOL";
+    public static final String ADAPTER_SCRIPT_EXASOL = "ADAPTER_SCRIPT_EXASOL";
+    public static final String TABLE_JOIN_1 = "TABLE_JOIN_1";
+    public static final String TABLE_JOIN_2 = "TABLE_JOIN_2";
```

Comment:
Are those relevant shared constants? If so, don't they indicate unnecessary coupling and low cohesion?